src/sdk/namespace_fs.js

```diff
@@ -399,17 +399,21 @@ const versions_dir_cache = new LRUCache({
     },
     validate: async ({ stat, ver_dir_stat }, { dir_path, fs_context }) => {
         const new_stat = await nb_native().fs.stat(fs_context, dir_path);
```

Comment:
this stat is also not guarded with try and catch, is there a situation where we could fail the whole list object versions because of that?

Reply:
this stat is for the bucket directory. so I don't think there is a situation like that. unless we delete the bucket and then we shouldn't even get to this point.